### PR TITLE
Remove duplicated indefinite articles 'a a' in char.cr doc

### PR DESCRIPTION
**Pull request by MakeNowJust**
It is bad English as far as I know.

### CODE REVIEW

File: src/char.cr [hunk 1]
```diff
@@ -288,7 +288,7 @@ struct Char
   #
   # The backslash character \ can be used to escape ^ or - and
   # is otherwise ignored unless it appears at the end of a range
-  # or the end of a a set.
+  # or the end of a set.
```
Comment (jhass) [inline src/char.cr]: Could even shorten this to "... the end of a range or set."

**Comment by straight-shoota** [comment]
merge?

**Comment by jhass** [comment]
Thank you!
